lgn-worker/src/manager.rs

```diff
@@ -65,6 +72,7 @@ impl ProversManager {
                 &config.public_params.groth16_assets.r1cs_file,
                 &config.public_params.groth16_assets.pk_file,
             )
+            .await
             .context("initializing Large prover")?;
```

Comment:
Making the download parallel should be trivial now, just need to add a `tokio::join!` here. I have not done that because of the memory use.

Reply:
> Making the download parallel should be trivial now

I think that in any case, S3 will be serving fast enough to saturate the line, so I don't think that would save wall-clock time.

Reply:
I agree, but I would rank all that quite low in priority terms, for re-starting a worker is (hopefully) not a very frequent operation.